resources/ec2_regional_config.go

```diff
@@ -0,0 +1,91 @@
+package resources
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func Ec2RegionalConfig() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_ec2_regional_config",
```

Comment:
as it uses ebs configs I suggest to use ebs somewhere in table name. Name is ok if it's purose to store all possible regional configs of ec2

Reply:
yes, the purpose is that if any more ec2 regional values show up we will store them there